docs/reference/deploy/master-config-reference.rst

```diff
@@ -1571,12 +1571,109 @@ The username for HTTP basic authentication (only allowed with ``type: basic``).
 
 The password for HTTP basic authentication (only allowed with ``type: basic``).
 
+**********
+ ``oidc``
+**********
+
+Applies only to Determined Enterprise Edition. The OIDC (OpenID Connect) configuration allows
+administrators to integrate an OIDC provider such as Okta for authentication in Determined and is
+used for :ref:`remote user <remote-users>` management.
+
+   For example:
+
+   .. code:: yaml
+
+      oidc:
+          enabled: true
+          provider: "Okta"
+          client_id: "xx0xx0"
+          client_secret: "xx0xx0"
+          idp_recipient_url: "https://determined.example.com"
+          idp_sso_url: "https://dev-00000000.okta.com"
+          authentication_claim: "string"
+          scim_authentication_attribute: "string"
+          auto_provision_users: true
+          groups_claim_name: "XYZ"
+          display_name_claim_name: "XYZ"
+
+``enabled``
+===========
+
+Whether to enable OIDC authentication. Defaults to ``false``.
+
+``provider``
+============
+
+The name of the OIDC provider. Officially supported: "okta".
+
+``client_id``
+=============
+
+The client identifier provided by the OIDC provider.
+
+``client-secret``
+=================
+
+The client secret provided by the OIDC provider. This should be kept confidential.
+
+``idp_recipient_url``
+=====================
+
+The URL where your IdP sends OIDC assertions.
+
+``idp_sso_url``
+===============
+
+The Single Sign-On (SSO) URL provided by the OIDC provider.
+
+``authentication_claim``
+========================
+
+The claim used for authentication in OIDC.
+
+``scim_authentication_attribute``
+=================================
+
+The attribute used for SCIM authentication.
+
+``auto_provision_users``
+========================
+
+Determines if users should be automatically created in Determined upon successful OIDC authentication.
+   -  ``true``: Automatic user provisioning is enabled.
+   -  ``false``: Automatic user provisioning is disabled.
+
+``groups_claim_name``
+=====================
+
+The claim name that specifies group memberships in OIDC.
+
+``display_name_claim_name``
+===========================
+
+The claim name from the OIDC provider used to set the user's display name in Determined.
+
 **********
  ``saml``
 **********
 
-Applies only to Determined Enterprise Edition. Specifies whether SAML SSO is enabled and the
-configuration to use it.
+Applies only to Determined Enterprise Edition. The SAML (Security Assertion Markup Language)
+configuration allows administrators to integrate a SAML provider such as Okta for authentication in
+Determined.
+
+For example:
+
+   .. code:: yaml
+
+      saml:
+          enabled: true
+          provider: "Okta"
+          idp_recipient_url: "https://determined.example.com/saml/sso"
+          idp_sso_url: "https://myorg.okta.com/app/...sso/saml"
+          idp_cert_path: "okta.cert"
+          auto_provision_users: true
```

Comment:
Could you add definitions for these options, similar to OIDC?

Reply:
@salonig23  added. please re-review.

Reply:
Looks great, thank you!